pyproject.toml

```diff
@@ -22,6 +22,7 @@ msrest = ">=0.7.1"
 typing-extensions = ">=3.7.4"
 aiohttp = { version = ">=3.0", optional = true }
 sphinx-rtd-theme = "^1.1.1"
+sphynx = "^0.0.3"
```

Comment:
I think this only needs to be a dev dependency since it's not required by anyone using the library. 

It can be moved using: `poetry add --dev sphynx`

Reply:
oh good call- will make a new pr. i also spelled it wrong